redfish-core/lib/processor.hpp

```diff
@@ -1358,10 +1688,63 @@ inline void requestRoutesProcessorCollection(App& app)
         asyncResp->res.jsonValue["@odata.id"] =
             "/redfish/v1/Systems/system/Processors";
 
-        collection_util::getCollectionMembers(
-            asyncResp,
-            boost::urls::url("/redfish/v1/Systems/system/Processors"),
-            processorInterfaces, "/xyz/openbmc_project/inventory");
+        crow::connections::systemBus->async_method_call(
```

Comment:
Use common function dbus::utility::getSubTreePaths()

Reply:
Will address this also as part of new commit (JIRA number will be attached)